app/server.go

```diff
@@ -151,24 +152,33 @@ func (a *App) StartServer() error {
 
 	if *a.Config().ServiceSettings.Forward80To443 {
 		if host, port, err := net.SplitHostPort(addr); err != nil {
-			l4g.Error("Unable to setup forwarding: " + err.Error())
+			mlog.Error("Unable to setup forwarding: " + err.Error())
 		} else if port != "443" {
 			return fmt.Errorf(utils.T("api.server.start_server.forward80to443.enabled_but_listening_on_wrong_port"), port)
```

Comment:
This error is returned to the StartServer execution from the command line, returned to the CLI to be showed in the command line, and at the same time written to the log as Critical. Probably I musn't be translated neither.

Reply:
Not really part of this PR. This isn't logged. Feel free to submit another pr for that.